cmd/cli/plugin/package/README.md

```diff
@@ -8,46 +8,65 @@ Package and Repository Operations ( Subject to Change ):
 
 ```sh
 >>> tanzu package --help
-Tanzu package management
+Tanzu package management (available, install INSTALLED_PACKAGE_NAME --package-name PACKAGE_NAME --version VERSION, installed, repository)
```

Comment:
This header for the help sounds a bit confusing

Reply:
It gets auto generated for all the commands based on the sub commands. I will try to figure out a way to make it more readable.